pkg/driver/docker/docker.go

```diff
@@ -90,9 +90,27 @@ func (d *dockerDriver) StartDevContainer(ctx context.Context, id string, labels
 }
 
 func (d *dockerDriver) DeleteDevContainer(ctx context.Context, id string, deleteVolumes bool) error {
-	// TODO: implement deleteVolumes
+	var volume string
 
-	return d.Docker.Remove(ctx, id)
+	if deleteVolumes {
+		// inspect container deleteVolumes
+		container, err := d.Docker.InspectContainers([]string{id})
+		if err != nil {
+			return err
+		}
+		volume = container[0].Config.Labels["dev.containers.id"]
```

Comment:
Is there a chance that Labels might be nil?

Alternatively, what if docker daemon misbehaves for some reason and len(containers) = 0?

Reply:
Done :+1: Thanks!